Rakefile

```diff
@@ -35,4 +36,8 @@ task :check_binstubs do
   end
 end
 
-task default: [:spec, :make_bin_executable, :yard, :rubocop, :check_binstubs]
+Kitchen::RakeTasks.new
+
+task integration: 'kitchen:all'
+
+task default: [:make_bin_executable, :yard, :rubocop, :check_binstubs, :integration]
```

Comment:
so since we run the tests on a [matrix of ruby versions](https://github.com/sensu-plugins/sensu-plugins-docker/blob/3.0.0/.travis.yml#L8-L13) we probably don't either want the default to execute all suites or we should modify travis to not execute the default task and do something like [this](https://github.com/sensu-plugins/sensu-plugins-dns/blob/2.0.0/.travis.yml#L22-L23)